lib/xcov/model/report.rb

```diff
@@ -81,5 +88,19 @@ def self.excluded_targets
       excluded_targets
     end
 
+    def self.included_targets
```

Comment:
Fair comment @thelvis4! However I'm not going to be strict on this regard given the time that it's taking to get stuff merge given my poor participation these days. Apologies guys 😬 

Merging this and releasing a new version afterwards.